explorer/tsconfig.json

```diff
@@ -16,7 +16,7 @@
     "esModuleInterop": true,
     "noImplicitAny": true,
     "baseUrl": ".",
-    "typeRoots": ["node_modules/@types", "@types"]
+    "typeRoots": ["node_modules/@types", "../node_modules/@types", "@types"]
```

Comment:
`ts-jest` doesn't support yarn workspaces. Need to configure install location of typescript